### PR TITLE
Flowerschaos interview

### PR DESCRIPTION
**Pull request by flowerschaos**

### CODE REVIEW

**Comment by flowerschaos** [comment]
summoning demons (lab completed)

File: interview/password.py [hunk 1]
```diff
@@ -16,4 +16,29 @@ def generate_password() -> str:
...
     # TODO implement generate_password function
     
-    return ''
\ No newline at end of file
+    import random
+
+    password = ''
+
+    # arrays
+    alphabetLower = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
```
Comment (bmazey) [inline interview/password.py:24]: nice use of list! 💯

File: interview/holiday.py [hunk 2]
```diff
@@ -9,4 +9,29 @@ def monthly_holiday(month: str) -> str:
 
     # TODO implement monthly_holiday function
 
-    return ''
+    if month == 'January':
```
Comment (bmazey) [inline interview/holiday.py:12]: 👌

File: interview/palindrome.py [hunk 3]
```diff
@@ -9,4 +9,4 @@ def is_palindrome(word: str) -> str:
 
     # TODO implement is_palindrome function
     
-    return True
+    return word == word[::-1]
```
Comment (bmazey) [inline interview/palindrome.py:12]: very nice but would appreciate a comment explanation! 👀

File: interview/rounder.py [hunk 4]
```diff
@@ -7,5 +7,9 @@ def round(number: float) -> int:
     """
 
     # TODO implement round function
-
-    return 0
\ No newline at end of file
+    if number % 1 >= .5:
```
Comment (bmazey) [inline interview/rounder.py:10]: great implementation -- looking for comments!

File: interview/shredder.py [hunk 5]
```diff
@@ -8,7 +8,7 @@ def shred_first_character(word: str) -> str:
 
     # TODO implement shred_first_character function
 
-    return ''
+    return word [1::]
```
Comment (bmazey) [inline interview/shredder.py:11]: spacing -> word[1::]

File: interview/shredder.py [hunk 6]
```diff
@@ -18,4 +18,4 @@ def shred_last_character(word: str) -> str:
 
     # TODO implement shred_last_character function
     
-    return ''
\ No newline at end of file
+    return word [:len(word)-1:]
```
Comment (bmazey) [inline interview/shredder.py:21]: spacing -> word[:len(word) - 1:]

**Review by bmazey — COMMENTED** [review]
108/100 excellent work! solid formatting, would like to see documentation (comments) on each implementation (-2) palindrome bonus (+10)

**Review by bmazey — APPROVED** [review]

**Pull request merged**